src/web/app/components/question-edit-form/question-edit-form.component.ts

```diff
@@ -15,7 +15,26 @@ import { SimpleModalType } from '../simple-modal/simple-modal-type';
 import { collapseAnim } from '../teammates-common/collapse-anim';
 import { QuestionEditFormMode, QuestionEditFormModel } from './question-edit-form-model';
 
-const CLEAN_PROPERTIES: Set<string> = new Set<string>(['isEditable', 'isCollapsed', 'isChanged']);
+const FEEDBACK_PATH_PROPERTIES: Set<string> = new Set<string>([
+  'giverType',
+  'recipientType',
+  'isUsingOtherFeedbackPath',
+  'numberOfEntitiesToGiveFeedbackToSetting',
+  'customNumberOfEntitiesToGiveFeedbackTo',
+]);
+const VISIBILITY_PROPTERIES: Set<string> = new Set<string>([
+  'isUsingOtherVisibilitySetting',
+  'showResponsesTo',
+  'showGiverNameTo',
+  'showRecipientNameTo',
+  'commonVisibilitySettingName',
+]);
+const CLEAN_PROPERTIES: Set<string> = new Set<string>([
+  ...VISIBILITY_PROPTERIES,
```

Comment:
I wouldn't put visibilities inside clean properties because clean properties represent UI properties that does not affect the question at the backend. Visibilities do change the question at the backend.